cmd/kubectl-gadget/collector.go

```diff
@@ -45,17 +36,8 @@ var processCollectorCmd = &cobra.Command{
 }
 
 var (
-	collectorParamLabel         string
-	collectorParamNode          string
-	collectorParamPodname       string
-	collectorParamContainername string
-	collectorParamAllNamespaces bool
-	collectorParamJsonOutput    bool
-	collectorParamThreads       bool
-)
-
-const (
-	GADGET_OPERATION = "gadget.kinvolk.io/operation"
+	collectorParams       utils.CommonFlags
```

Comment:
I was thinking to have several commands implemented in the same file in a similar way to cmd/kubectl-gadget/bcck8s.go, so that the code is factorised. But if we start using the `utils` package, factorization can be achieved regardless, so I don't have a strong opinion...